src/components/ContractWriteResultCard/ContractWriteResultCard_View.tsx

```diff
@@ -4,13 +4,19 @@ type Props = {
   hash?: string;
   isSuccess: boolean;
   errorMsg?: string;
+  txType: "bet" | "deposit" | "withdrawal" | undefined;
```

Comment:
Could we please make these an enum?

Search for `enum FilterOptions` in the FE if you need an example